src/main/java/seedu/address/model/person/Appointment.java

```diff
@@ -0,0 +1,146 @@
+package seedu.address.model.person;
```

Comment:
Can move to either the parent package `model` or to a new package `model.appointment`